airflow-core/src/airflow/api_fastapi/core_api/routes/public/dag_run.py

```diff
@@ -73,6 +74,8 @@
 from airflow.utils.state import DagRunState
 from airflow.utils.types import DagRunTriggeredByType, DagRunType
 
+log = structlog.get_logger()
```

Comment:
Can we associate the logger with the module? Easier imo to comprehend that way
```
log = structlog.get_logger(__name__)
```

Reply:
we could but there are already many many instances like this in the codebase. we can make it uniform in a followup.

Reply:
Alright thats ok

Reply:
PR for cleanup https://github.com/apache/airflow/pull/48480